collections/_infographics/emissions/emissions-czechia.md

```diff
@@ -7,7 +7,29 @@ published:     2022-07-26
+* __Combustion in the industry:__ 9.96 million tonnes of CO<sub>2</sub> (7.7% of total emissions, i.e. 940 kg CO<sub>2</sub>eq per capita per year). Emissions from the combustion of fossil fuels for industrial use, typically when something needs to be heated during production (distillation, drying, melting, etc.).
+* __Domestic, institutional and agricultural combustion:__ 13.15 million tonnes of CO<sub>2</sub> (10.2% of total emissions, i.e. 1240 kg CO<sub>2</sub>eq per capita per year). This sector includes heating and hot water (unless a heating plant supplies the energy), cooking, etc. It also includes fuel combustion in agriculture and forestry.
+* __Waste management:__ 5.7 million tonnes CO<sub>2</sub>eq per year (4.4% of total emissions, 540 kg CO<sub>2</sub>eq per capita per year). Emissions from waste management are mainly produced by landfills, which release methane into the atmosphere from the decomposition of biodegradable material in the landfill body. The solution is to ban landfilling of usable waste, following the example of most EU countries, and to use biodegradable waste to produce biomethane instead of natural gas, for example, in transport.
+* __Agriculture:__ 8.61 million tonnes CO<sub>2</sub>eq (6.7% of total emissions, 810 kg CO<sub>2</sub>eq per capita per year). Agricultural emissions come mainly from livestock farming (4.1 million tonnes) in the form of methane emissions and from tillage and associated N2O emissions (4.23 million tonnes CO<sub>2</sub>eq). Reducing the number of livestock (and the associated reduction in beef and dairy consumption), proper management of manure (e.g. by stabilising it in biogas plants) and less intensive fertiliser application would help to reduce methane emissions from agriculture.
+
+To make it easier to compare emissions <!--[-->across EU countries<!--](/infografiky/emise-vybrane-staty)-->, we omit the category of _Land use, land use change and forestry_ (_LULUCF_). Due to carbon storage in green areas, this category has negative emissions in most EU countries, which complicates visualisation. The LULUCF sector is also often [omitted](https://climateactiontracker.org/methodology/indc-ratings-and-lulucf/) from comparisons because it contains high uncertainty in the data. Negative values can mask _structural_ emissions from energy, industry and agriculture, and it is more prone to fluctuations over time. In particular, the Czech Republic has seen a significant fluctuation in recent years due to massive logging during the bark beetle calamity. For 2018, emissions in this sector were estimated to be _positive_ at 5.8 Mt CO<sub>2</sub>eq.
```

Comment:
Open question here: LULUCF's https://climateactiontracker.org/methodology/indc-ratings-and-lulucf appears to no longer be available. Should we follow the redirect they have on their page or if there a better URL?